src/main/scala/no/ndla/draftapi/DraftApiProperties.scala

```diff
@@ -53,13 +60,17 @@ object DraftApiProperties extends LazyLogging {
   val RunWithSignedSearchRequests = propOrElse("RUN_WITH_SIGNED_SEARCH_REQUESTS", "true").toBoolean
   val DraftSearchIndex = propOrElse("SEARCH_INDEX_NAME", "draft-articles")
   val ConceptSearchIndex = propOrElse("CONCEPT_SEARCH_INDEX_NAME", "draft-concepts")
+  val AgreementSearchIndex = propOrElse("AGREEMENT_SEARCH_INDEX_NAME", "draft-agreements")
   val DraftSearchDocument = "article-drafts"
+  val AgreementSearchDocument = "agreement-drafts"
   val ConceptSearchDocument = "concept-drafts"
   val DefaultPageSize = 10
   val MaxPageSize = 100
   val IndexBulkSize = 200
   val ElasticSearchIndexMaxResultWindow = 10000
 
+  final val allowedAuthorsStr = "originator,photographer,artist,editorial,writer,scriptwriter,reader,translator,director,illustrator,cowriter,composer,processor,facilitator,editorial,linguistic,idea,compiler,correction,rightsholder,publisher,distributor,supplier"
```

Comment:
Foreslår å legge denne sammen med klassen hvor den blir brukt (siden den er kun for dokumentasjon. Tror heller ikke den trenger å være `final`?

Reply:
Måtte visst være final om jeg skulle bruke den i det feltet, men legger den rett inn istedet.